docs/FAQ.md

```diff
@@ -120,4 +125,4 @@ You can stop lending when you choose. Withdrawing early may result in a loss of
 The portfolio value is calculated as `(fyDai * APR) / (days to maturity / 365)`
```

Comment:
Portfolio value (at maturity) is the fyDai balance which represents the amount of Dai that the fyDai can be redeemed for at maturity.